lib/tasks/users.rake

```diff
@@ -33,4 +33,29 @@ namespace :users do
     remaining = User.where.not(latitude: nil, longitude: nil).where(state: nil).count
     p "#{remaining} users left to be updated.  Task can be reran in 24 hours."
   end
+
+  desc "Tags select users as Community Leaders"
+  task seed_leader: :environment do
+    users = [ {id: 4, first_name: 'Nell', last_name: 'Shamrell-Harrington' }, {id: 3, first_name: 'Rick', last_name: 'Rein'}]
+    users_id = User.where(id: [3,4])
```

Comment:
As these are records, perhaps rename this variable to something like `user_records`, since you are using `users` above.